.github/workflows/ci-model-regression-on-schedule.yml

```diff
@@ -419,7 +419,7 @@ jobs:
           # Add this issue to a project column
           curl \
             -X POST \
-            -s -H "Authorization: token ${{ secrets.RASABOT_ASSIGN_ISSUE_RESEARCH_BOARD_TOKEN }}" \
+            -s -H "Authorization: token ${{ secrets.RASABOT_ASSIGN_ISSUES_BOARD_TOKEN }}" \
```

Comment:
We could possibly move the assignment of the issue to the new `assign-default-project` workflow, it may make things more modular

Reply:
up to you in my opinion. I think both has it's advantages. Your approach is more DRY while the current approach keeps things together which makes it easier to understand a workflow without understand all workflows). Maybe it's worth adding a comment where the issue to project mapping is done in this workflow in case you decide to move it out?

Reply:
yeah I agree with you, both ways of doing it are acceptable. I wanted get @tczekajlo opinion on this before moving it (or not).

I was thinking that we'll likely have other rules in the future to automate project assignment, and keeping all of these rules in one place might make things more maintainable 🤔

Reply:
The new `assign-default-project` workflow looks good to me 😄  The only thing that has to be changed before moving it is the label that is used for the issues created for the research team, now it uses the "bug" label that is quite common, I guess we could add an additional label like 'model-regression-tests:performance-drop` (?) in order to distinguish the issues

Reply:
or something like `tool:model-regression-tests`? That'd make it consistent with other tools we use elsewhere (even if this one is in-house ^^)

Reply:
@tczekajlo I've made the change in this commit 1e114bad245 and also added the label to existing issues. Can I get a review? 🤩